packages/composites/lido/src/endpoint/exchangerate.ts

```diff
@@ -0,0 +1,39 @@
+import { Validator, Requester } from '@chainlink/ea-bootstrap'
+import { AdapterContext, AdapterRequest, AdapterResponse } from '@chainlink/types'
+//import { makeExecute as makeViewFunction } from '../../../../sources/view-function'
+import { makeExecute as makeViewFunction } from '@chainlink/view-function-adapter'
+import type { Config } from '../config/index'
+
+import abi from '../abi/WstETH.json'
+
+export const supportedEndpoints = ['exchangerate']
+
+export const execute = async (
```

Comment:
If you use the `Execute type` it will inherit the types on the input parameters and return.